src/scripts.js

```diff
@@ -383,6 +399,20 @@ function displayIngredientsNeeded(recipe) {
   }
 }
 
+async function removeRecipeIngredients() {
+  for (const ingredient of selectedRecipe.ingredients) {
```

Comment:
We were struggling with trying to put await in a forEach. We aren't quite understanding exactly why (and we're thinking we might try to still re-work this await), but using this syntax works.